retrofit/src/main/java/retrofit2/OkHttpCall.java

```diff
@@ -140,19 +139,19 @@ public void enqueue(final Callback<T> callback) {
         }
       }
     }
-
+    
     if (failure != null) {
       callback.onFailure(this, failure);
       return;
     }
-
+    
     if (canceled) {
       call.cancel();
     }
-
     call.enqueue(
-        new okhttp3.Callback() {
-          @Override
+      new okhttp3.Callback() {
+        @Override
+        @SuppressWarnings("calledmethods:required.method.not.called") //Response<T> response will go out scope through callback.onResponse() does not take responsiblity for closing response and the resource will still be active. This is also anonymous class therefore there is no way to alias the constructor and response .//https://javadoc.io/doc/com.squareup.okhttp3/okhttp/3.14.9/okhttp3/Callback.html
```

Comment:
Is this or is this not a false positive? The comment doesn't explicitly say, but seems to imply that you think this is a leak? But, you also mention generics, so I'm not sure whether you're arguing this is a false positive or a true leak.